### PR TITLE
Added CopyrightInfo component

### PR DESCRIPTION
**Pull request by Hudayberdiev2000**
All sizes are taken from figma,

### CODE REVIEW

**Comment by Hudayberdiev2000** [comment]
i wanted to create another pull request, for SpecialMealCard component, But it is added to this pull request.

**Comment by Hudayberdiev2000** [comment]
also fixed the bugs that you told me in previous pull request(CopyrightInfo component)

File: src/components/atoms/specialMealCard/SpecialMealCard.module.scss
Comment (bruno-keiko) [inline src/components/atoms/specialMealCard/SpecialMealCard.module.scss:1]: Please remove any unrelated changes to this issue.

File: src/components/atoms/specialMealCard/SpecialMealCard.stories.tsx
Comment (bruno-keiko) [inline src/components/atoms/specialMealCard/SpecialMealCard.stories.tsx:1]: Please remove any unrelated changes to this issue.

**Comment by bruno-keiko** [comment]
> i wanted to create another pull request, for SpecialMealCard component, But it is added to this pull request.

You can create a separate branch for other issues and revert unrelated changes in this one.

**Review by bruno-keiko — APPROVED** [review]
